client/my-sites/checkout/checkout-thank-you/style.scss

```
@@ -307,3 +307,307 @@
+		height: 220px;
+		width: 220px;
+		top: -110px;
+		left: calc( 50% + 110px );
+		animation-delay: 1s;
+	}
+	.gridicons-phone {
+		height: 80px;
+		width: 80px;
+		top: 130px;
+		left: calc( 50% - 390px );
+		animation-delay: 6s;
+	}
+	.gridicons-reader {
+		height: 130px;
+		width: 130px;
+		top: 360px;
+		left: calc( 50% - 170px );
+		transform: rotate( 12deg );
+		animation-delay: 8s;
+	}
+	.gridicons-star {
+		height: 90px;
+		width: 90px;
+		top: 380px;
+		left: calc( 50% + 60px );
+	}
+	.gridicons-video {
+		top: 245px;
+		left: calc( 50% - 160px );
+		animation-delay: 3s;
+	}
+	.gridicons-align-image-right {
+		top: 280px;
+		left: calc( 50% + 90px );
+		animation-delay: 7s;
+	}
+	.gridicons-bookmark {
+		top: 460px;
+		height: 110px;
+		width: 110px;
+		left: calc( 50% - 440px );
+		animation-delay: 5s;
+	}
+	.gridicons-briefcase {
+		top: 320px;
+		height: 80px;
+		width: 80px;
+		left: calc( 50% - 450px );
+		animation-delay: 1s;
+	}
+	.gridicons-calendar {
+		top: 460px;
+		left: calc( 50% + 300px );
+		height: 80px;
+		width: 80px;
+		animation-delay: 2s;
+	}
+	.gridicons-clipboard {
+		top: 240px;
+		left: calc( 50% + 290px );
+		height: 50px;
+		animation-delay: 6s;
+	}
+	.gridicons-cloud-upload {
+		top: 180px;
+		left: calc( 50% - 280px );
+		height: 40px;
+		width: 40px;
+		animation-delay: 5s;
+	}
+	.gridicons-cog {
+		top: 110px;
+		left: calc( 50% + 160px );
+		height: 40px;
+		width: 40px;
+		animation-delay: 1s;
+	}
+	.gridicons-customize {
+		top: 240px;
+		height: 40px;
+		width: 40px;
+		left: calc( 50% - 400px );
+		animation-delay: 1s;
+	}
+	.gridicons-help {
+		top: 460px;
+		left: calc( 50% + 160px );
+		height: 90px;
+		width: 90px;
+		animation-delay: 1s;
+	}
+	.gridicons-link {
+		top: 60px;
+		left: calc( 50% - 160px );
+		height: 50px;
+		width: 50px;
+		transform: rotate( -45deg );
+		animation-delay: 5s;
+	}
+	.gridicons-lock {
+		top: 60px;
+		left: calc( 50% - 400px );
+		height: 50px;
+		width: 50px;
+		animation-delay: 6s;
+	}
+	.gridicons-pages {
+		top: 70px;
+		left: calc( 50% + 70px );
+		height: 50px;
+		width: 50px;
+		animation-delay: 8s;
+	}
+	.gridicons-share {
+		top: 280px;
+		left: calc( 50% + 360px );
+		animation-delay: 2s;
+	}
+	.gridicons-stats {
+		top: 470px;
+		left: calc( 50% - 270px );
+		height: 80px;
+		width: 80px;
+		animation-delay: 9s;
+	}
+}
+
+.checkout-processing-screen__content {
+	position: relative;
+	z-index: 1;
+	max-width: 500px;
+	margin: 0 auto;
+	padding-bottom: 30px;
+	color: var( --color-white );
+}
+
+/* Title */
+@keyframes fade-in {
+	0% {
+		transform: translateY( 0 );
+		opacity: 0;
+	}
+	100% {
+		transform: translateY( -80px );
+		opacity: 1;
+	}
+}
+
+.is-section-checkout-thank-you .empty-content__title {
+	font-size: 20px;
+	font-weight: normal;
+}
+
+.is-section-checkout-thank-you .empty-content__line {
+	font-size: 16px;
+	font-weight: normal;
+}
+
+.checkout-thank-you__wordpress-logo {
+	height: 180px;
+	width: 180px;
+	fill: var( --color-primary );
```

I fear that this means the logo colour will change with difference schemes, when something similar to #30683 should happen instead. I've not tested this, and I might be wrong anyway though. :)

```suggestion
	fill: var( --color-wpcom );
```

---

Ohhhh I was wondering about that. With the new Color Studio updates I wasn't sure what the value was. I also need to check to make sure the background and surrounding elements don't also change with the color schemes (I don't think they do...) Thanks @Aurorum ! Will fix shortly.